plugins/inputs/opcua/README.md

```diff
@@ -39,6 +39,10 @@ to use them.
   #
   ## Maximum time allowed for a request over the established connection.
   # request_timeout = "5s"
+
+  # Maximum time that a session shall remain open without activity.
+  # Default is 20 minutes.
+  # session_timeout = "60s"
```

Comment:
The commented option should display the default. This is not the case here, so you should do
```suggestion
  # session_timeout = "20m"
```

Reply:
done